product_product_template_link/static/description/index.html

```diff
@@ -0,0 +1,432 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
this line should be removed: `DeprecationWarning: XML declarations in HTML module descriptions are deprecated since Odoo 17, product_product_template_link can just have a UTF8 description with not need for a declaration.`